src/snowflake/snowpark/functions.py

```diff
@@ -6702,17 +6702,75 @@ def sproc(
         registered. Invoking :func:`sproc` with ``replace`` set to ``True`` will overwrite the
         previously registered function.
 
+        4. To describe the return type for a stored procedure that returns tabular data, use
+        one of the following ways:
+
+            - (Recommended) Describe the return type using :attr:`~snowflake.snowpark.types.StructType`
+              and :attr:`~snowflake.snowpark.types.StructField`. Set ``return_type =
+              StructType([StructField("a", DataTypeA()), ...])`` to describe the case
+              ``RETURNS TABLE(A DataTypeA, ...)``.
+
+            - Set ``return_type = StructType([])`` to describe the case ``RETURNS TABLE()``.
+
+            - When using type hints, the return type of function can be set as
+              :attr:`~snowflake.snowpark.types.SnowparkDataFrame`. This registers a table
+              stored procedure with return type defined using ``RETURNS TABLE()``.
+
     See Also:
         :class:`~snowflake.snowpark.stored_procedure.StoredProcedureRegistration`
 
-    Example::
+    Example 1::
         >>> from snowflake.snowpark.types import IntegerType
         >>> @sproc(return_type=IntegerType(), input_types=[IntegerType(), IntegerType()], packages=["snowflake-snowpark-python"])
         ... def add_sp(session_, x, y):
         ...     return session_.sql(f"SELECT {x} + {y}").collect()[0][0]
         ...
         >>> add_sp(1, 1)
         2
+
+    Example 2::
```

Comment:
are these example same as ones in stored_procedures.py? If the same, then we don't need them here? Just refer to examples in stored_procedures.py?

Reply:
They are already referred in See Also so I'll just revert this section.